python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -794,6 +825,35 @@ def make_channel(
             interval=self.interval,
         )
 
+    def get_opened_extras(self) -> PipesExtras:
+        # Extract the cluster log location from the SparkSession. This requires
+        # digging into the databricks `clusterUsageTags` config. This is set
+        # automatically by Databricks in the spark session that is created
+        # prior to job execution. Here are some sparse docs on cluster log
+        # delivery:
+        #   https://docs.databricks.com/en/clusters/configure.html#cluster-log-delivery
+        #
+        # It is not clear whether official docs exist for the config set in
+        # `spark.databricks.clusterUsageTags`, but you can see the full spark config for a job by
+        # selecting the "Spark UI" tab and then "Environment" on the job details page in the
+        # Databricks UI.
+        from py4j.protocol import Py4JJavaError
+        from pyspark.sql import SparkSession
+
+        spark = SparkSession.getActiveSession()
+        assert spark is not None, "Expected an active SparkSession"
```

Comment:
I think you should throw `DagsterPipesError` if none.

I don't believe in ever using asserts since users can theorectically turn them off.